src/helpers/misc.ts

```diff
@@ -129,7 +129,7 @@ export function getDefaultCellHeight(cell: Cell | undefined): Pixel {
     return DEFAULT_CELL_HEIGHT;
   }
   const fontSize = computeTextFontSizeInPixels(cell.style);
-  const multiLineText = cell.content.split(NEWLINE);
+  const multiLineText = cell.isFormula ? [cell.content] : cell.content.split(NEWLINE);
```

Comment:
then the array contains a single element which is a multiline text. That's weird because it's supposed to contain only single lines. It works because it's only used for `.length`, but still, it's weird.
I think I would just hard-code `1`.
Maybe with a comment
```suggestion
  // the number of lines should be computed from the formula result, but it's not evaluated at this point
  const numberOfLines = cell.isFormula ? 1 : cell.content.split(NEWLINE).length;
```